roles/openshift_logging_elasticsearch/vars/main.yml

```diff
@@ -10,3 +10,6 @@ es_min_masters_default: "{{ (openshift_logging_elasticsearch_replica_count | int
 es_min_masters: "{{ (openshift_logging_elasticsearch_replica_count == 1) | ternary(1, es_min_masters_default) }}"
 es_recover_after_nodes: "{{ openshift_logging_elasticsearch_replica_count | int }}"
 es_recover_expected_nodes: "{{ openshift_logging_elasticsearch_replica_count | int }}"
+es_masters_quorum: "{{ openshift_logging_elasticsearch_node_topology.masters.replicas |int /2 + 1 }}"
```

Comment:
This is fundamentally the same as es_min_masters, shouldnt we reuse.  I actually think we can remove es_min_masters.